src/ontology/mondo-ingest.Makefile

```diff
@@ -159,6 +159,17 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+$(COMPONENTSDIR)/icd11foundation.owl: $(TMPDIR)/icd11foundation_relevant_signature.txt | component-download-icd11foundation.owl
```

Comment:
## Build check passing
I reran and the following things I was trying to fix are now addressed:
- `slurp/icd11foundation.tsv` appears
- `lexmatch/unmapped_icd11foundation_lex.tsv` appears (among many other lexmatch files) and has over 1k lines (~6k)

As of my writing this, the build is still running. But I'll push again when the rest of it completes.

Reply:
@matentzn I reviewed this with Trish today. These two conditions checked above have been met for a couple weeks, and these are the conditions needed to be met to address this as success.

We're not merging this ("data PRs") for a variety of reasons; standard part of the mondo-ingest developer workflow.

FYI see how long it took me to run above, per your request.